src/Microsoft.PowerShell.Commands.Utility/commands/utility/DebugRunspaceCommand.cs

```diff
@@ -101,19 +101,11 @@ public Guid InstanceId
         }
```

Comment:
@KirkMunro, your last commit had 1 failures in `PowerShell-CI-windows`
Invoke-WebRequest tests.Denial of service.Charset Parsing
```powershell
Expected the actual value to be greater than 10, but got 7.52063586097946.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\WebCmdlets.Tests.ps1: line 1925
1925:             $pathologicalRatio | Should -BeGreaterThan 10
```